templates/paragraphs/paragraph--oe-description-list.html.twig

```diff
@@ -0,0 +1,19 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'Description list' type paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
```

Comment:
I'm not sure I understand what you mean. The template we extend in paragraphs comes from the Paragraphs module which is a contrib module, is not ours. Our module is oe_paragraphs.
The same applies for the other comment related to the template.

Reply:
we extend the paragraphs base template in @oe_theme/paragraphs/paragraph.html.twig, was saying since we do that we should extend that one for variants.
But since we dont for the others neither I guess we can consider our paragraph.html.twig as a fallback